docs/netdata-agent/versions-and-platforms.md

```diff
@@ -0,0 +1,70 @@
+|       Alma Linux        |             8.x, 9.x             |               `x86_64`, `AArch64`                |     RPM      |
+|      Amazon Linux       |             2, 2023              |               `x86_64`, `AArch64`                |     RPM      |
+|         Centos          |               7.x                |                     `x86_64`                     |     RPM      |
+|         Debian          |         10.x, 11.x, 12.x         |       `x86_64`, `i386`, `ARMv7`, `AArch64`       |     DEB      |
+|         Fedora          |            37, 38, 39            |               `x86_64`, `AArch64`                |     RPM      |
+|        OpenSUSE         | Leap 15.4, Leap 15.5, Tumbleweed |               `x86_64`, `AArch64`                |     RPM      |
+|      Oracle Linux       |             8.x, 9.x             |               `x86_64`, `AArch64`                |     RPM      |
+| Redhat Enterprise Linux |               7.x                |                     `x86_64`                     |     RPM      |
+| Redhat Enterprise Linux |             8.x, 9.x             |               `x86_64`, `AArch64`                |     RPM      |
+|         Ubuntu          |       20.04, 22.04, 23.10        |            `x86_64`, `i386`, `ARMv7`             |     DEB      |
+
+> IMPORTANT: Linux distributions frequently provide binary packages of Netdata. However, the packages you will find at the distributions' repositories may be outdated, incomplete, missing significant features or completely broken. We recommend to use the packages we provide. 
+
+## Third party Supported Binary Packages
+
+The following distributions always provide the latest stable version of Netdata:
+
+|  Platform  | Platform Versions |    Released Packages Architecture    |
+|:----------:|:-----------------:|:------------------------------------:|
+| Arch Linux |      Latest       | All the Arch supported architectures |
+| MacOS Brew |      Latest       | All the Brew supported architectures |
+
+
+## Builds from Source
+
+We guarantee Netdata builds from source for the platforms we provide automated binary packages. These platforms are automatically checked via our CI, and fixes are always applied to allow merging new code into the nightly versions.
+
+The following builds from source should usually work, although we don't regularly monitor if there are issues:
+
+|              Platform               |     Platform Versions      |
+|:-----------------------------------:|:--------------------------:|
+|         Linux Distributions         | Latest unreleased versions |
+|       FreeBSD and derivatives       |         13-STABLE          |
+|       Gentoo and derivatives        |           Latest           |
+|     Arch Linux and derivatives      |      latest from AUR       |
+|                MacOS                |         11, 12, 13         |
+| Linux under Microsoft Windows (WSL) |           Latest           |
```

Comment:
We _DO NOT SUPPORT WSL_ at all. Period. End of story.

It is not suitable for production usage, and we should not be making any comments encouraging users to use it in any way.